app/uk/gov/hmrc/cdsimportsddsfrontend/controllers/SubmitDeclarationController.scala

```diff
@@ -54,7 +53,7 @@ class SubmitDeclarationController @Inject()(submitTemplate: submit_declaration,
       },
       validatedForm => {
         declarationStore.deleteAllNotifications()
-        declarationService.submit(request.user.eori, validatedForm.declarationXml)
+        declarationService.submit(request.user.eori, validatedForm.declarationXml, None)
```

Comment:
i realise this will eventually go away, but can we extract the FunctionReferenceID from the XML here, so we store the actual LRN instead of nothing at all?

Reply:
yes, agree. I shouldn't have hardcoded `None` there